cmake/MBedExecutable.cmake

```diff
@@ -46,34 +46,45 @@ function(add_mbed_executable EXECUTABLE)
 	target_link_libraries(${EXECUTABLE} mbed-os)
 
 	set(BIN_FILE ${CMAKE_CURRENT_BINARY_DIR}/${EXECUTABLE}.bin)
+	set(HEX_FILE ${CMAKE_CURRENT_BINARY_DIR}/${EXECUTABLE}.hex)
 
 
 	if("${MBED_TOOLCHAIN_NAME}" STREQUAL "ARMC6")
 		# the ArmClang CMake platform files automatically generate a memory map
 		target_link_libraries(${EXECUTABLE} --info=totals --map)
 		set(MAP_FILE ${CMAKE_CURRENT_BINARY_DIR}/${EXECUTABLE}${CMAKE_EXECUTABLE_SUFFIX}.map)
 
-		set(OBJCOPY_COMMAND ${OBJCOPY_EXECUTABLE} --bin -o ${BIN_FILE} $<TARGET_FILE:${EXECUTABLE}>)
+		set(OBJCOPY_ELF_TO_BIN_COMMAND ${OBJCOPY_EXECUTABLE} --bin -o ${BIN_FILE} $<TARGET_FILE:${EXECUTABLE}>)
+		set(OBJCOPY_ELF_TO_HEX_COMMAND ${OBJCOPY_EXECUTABLE} --i32 -o ${HEX_FILE} $<TARGET_FILE:${EXECUTABLE}>)
 
 	elseif("${MBED_TOOLCHAIN_NAME}" STREQUAL "GCC_ARM")
 
 		set(MAP_FILE ${CMAKE_CURRENT_BINARY_DIR}/${EXECUTABLE}.map)
+		set(HEX_FILE ${CMAKE_CURRENT_BINARY_DIR}/${EXECUTABLE}.hex)
```

Comment:
I don't think this line is doing anything since it's already in the block above.

Reply:
you're right! I've removed the duplicate.